assets/js/blocks/attribute-filter/style.scss

```diff
@@ -67,8 +67,8 @@
 	}
 
 	.wc-blocks-components-form-token-field-wrapper .components-form-token-field__input-container {
-		border: 1px solid $gray-700;
-		border-radius: 4px;
+		border: 0;
+		width: 85%;
```

Comment:
Given that the chevron occupies 30px and is positioned 20px from the left, I wonder if we could use `calc(100% - 50px)` instead of a percentage width. I think it would be more accurate.

Reply:
Great suggestion, I didn't know about that 😅 Thanks!